From 059b517f9ef6cbdc696e0983ce255b1728042827 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Thu, 25 Aug 2022 16:46:04 +0800
Subject: [PATCH] smbtorture: skip test case tfork_cmd_send

The test case tfork_cmd_send fails on target as it requires a script
located in the source directory:

$ smbtorture ncalrpc:localhost local.tfork.tfork_cmd_send
test: tfork_cmd_send
/buildarea/build/tmp/work/core2-64-poky-linux/samba/4.14.14-r0/samba-4.14.14/testprogs/blackbox/tfork.sh:
Failed to exec child - No such file or directory

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 lib/util/tests/tfork.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/lib/util/tests/tfork.c b/lib/util/tests/tfork.c
index 70ae975..4826ce6 100644
--- a/lib/util/tests/tfork.c
+++ b/lib/util/tests/tfork.c
@@ -839,10 +839,6 @@ struct torture_suite *torture_local_tfork(TALLOC_CTX *mem_ctx)
 				      "tfork_threads",
 				      test_tfork_threads);
 
-	torture_suite_add_simple_test(suite,
-				      "tfork_cmd_send",
-				      test_tfork_cmd_send);
-
 	torture_suite_add_simple_test(suite,
 				      "tfork_event_file_handle",
 				      test_tfork_event_file_handle);
-- 
2.25.1

